docs/server/source/quickstart.md

```diff
@@ -16,12 +16,12 @@ C. Assign rwx(read/write/execute) permissions to the user for default database d
 $ sudo chmod -R 700 /data/db
 ```
 
-D. Run MongoDB:
+D. Run MongoDB (do not close this terminal):
```

Comment:
I would add a "but" before "do not close this terminal just so that it reads smoothly, i.e.
```text
D. Run MongoDB (but do not close this terminal):
```